www/main/components/addons/bani-controller/components/BaniController.jsx

```diff
@@ -0,0 +1,164 @@
+import { useStoreState, useStoreActions } from 'easy-peasy';
```

Comment:
The directory path for this file seems weird. 
Personally I would 
- have a folder for each "page" (settings, search, toolbar, etc.)
- a components folder for common components
- a util folder for anything that just helper functions.

Reply:
Using the structure we discussed in slack.